.github/workflows/pack_and_release.yml

```
@@ -32,4 +32,4 @@ jobs:
         run: dotnet pack -v normal -c Release --include-source -p:Version=${{steps.get-version.outputs.version}} -o nupkg ROSNET/ROSNET.*proj
 
       - name: Push package to NuGet
-        run: dotnet nuget push /home/runner/work/ROSNET/ROSNET/nupkg/ROSNET.${{steps.get-version.outputs.version}}.nupkg --api-key ${{secrets.NUGET_API_KEY}} --source https://api.nuget.org/v3/index.json --skip-duplicate
+        run: dotnet nuget push /home/runner/work/ROSNET/ROSNET/nupkg/RosNet.${{steps.get-version.outputs.version}}.nupkg --api-key ${{secrets.NUGET_API_KEY}} --source https://api.nuget.org/v3/index.json --skip-duplicate
```

```suggestion
        run: dotnet nuget push ROSNET/nupkg/RosNet.${{steps.get-version.outputs.version}}.nupkg --api-key ${{secrets.NUGET_API_KEY}} --source https://api.nuget.org/v3/index.json --skip-duplicate
```
This should probably use GitHub's puget-registry, see https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry